script/packtpub.py

```diff
@@ -124,6 +124,10 @@ def download_ebooks(self, types):
             for type in types]
 
         directory = self.__config.get('path', 'path.ebooks')
+
+        if self.__config.get('path', 'path.separate') == "true":
+            directory = directory + '/' + self.info['title'] + ' - ' + self.info['author']
```

Comment:
On unix system usually is desiderable to avoid empty space, if you prefer you can have also
```
path.separate.space=_
```
as default, so is configurable